docs/articles/introduction-to-IoT-security.md

```diff
@@ -0,0 +1,117 @@
+# Introduction to IoT security
+
+IoT systems can contain many moving parts, each of which need to communicate and coordinate with each other in a secure manner. In a highly connected environment, security is not simply a matter of protecting individual resources, but is also concerned with protecting the information that is transferred between those resources.
+
+In an IoT solution, security and safety are commonly intertwined. In a business environment, data emanating from business processes is commercially valuable and is often a target for large-scale industrial espionage. Software developers have traditionally been more concerned with hardening systems to make the action of mounting an attack commercially unattractive (it can cost a third party more in resources to break into the system and steal information than is profitable). Software engineers constructing IoT solutions have much broader security concerns. Many IoT systems control critical operations and the core of industrial and civil infrastructure. These are prime targets for politically motivated attacks (terrorism)rather than those driven by profit. Consequently, digital security will be increasingly interwoven with the physical safety of life and equipment. This should provide deep motivation for ensuring that security is addressed in a robust and thorough manner.  
```

Comment:
I like the opening theme of this paragraph: how security and safety are intertwined concerns. However, the rest of the paragraph quickly diverges.

I also think that our examples tend toward things that people already view as critical (public services, infrastructure, etc.) Part of the surprise for me in researching IoT and security is that seemingly innocuous systems can present safety concerns (e.g. sending a malicous document to a printer and starting a fire).

I'd like to hook the reader quickly with some kind of "here's some things that might surprise you".